release-notes-3.0.0.md

```diff
@@ -6,6 +6,7 @@ ${version-number}
 #### New Features
 - [CAF-3518](https://jira.autonomy.com/browse/CAF-3518): Use latest worker-document framework.  
   Uses the latest worker-document framework which includes support for executing a post processing script against the received document.
+- [SCMOD-3559](https://jira.autonomy.com/browse/SCMOD-3559): Integrates setting of custom values by environment variables for the features of the document worker framework for loading the object code of the static/dynamic scripts.
```

Comment:
I don't think you need this. It's part of the feature described by CAF-3519, the changes to worker-document framework to support scripting.